### PR TITLE
Bug fix in CLI-plugins

### PR DESCRIPTION
**Pull request by swapnil-bobade27**
Signed-off-by: Swapnil Bobade <swapnil.bobade@ibm.com>

### CODE REVIEW

File: playbooks/roles/ocp-cli-plugins/tasks/oc-mirror-cli-plugin.yml [hunk 1]
```diff
@@ -22,6 +22,13 @@
         version: main
       when: oc_mirror_path.stat.exists == false
 
+    - name: Updating Dockerfile with Power specific go version
+      replace:
+          path: ~/plugins/oc-mirror/Dockerfile
+          regexp: "amd64"
+          replace: "ppc64le"
+      register: check_go_version
```
Comment (yussufsh) [inline playbooks/roles/ocp-cli-plugins/tasks/oc-mirror-cli-plugin.yml:30]: Any use of `check_go_version`?

Comment (swapnil-bobade27) [reply playbooks/roles/ocp-cli-plugins/tasks/oc-mirror-cli-plugin.yml]: Removed the unused vars

File: playbooks/roles/ocp-cli-plugins/tasks/oc-mirror-cli-plugin.yml [hunk 2]
```diff
@@ -33,11 +40,14 @@
...
 
-    - name: Run the make build command from oc-mirror repository
-      make:
+    - name: Execute the Podman build command
+      shell: podman build -f Dockerfile -t local/go-toolset .
+      args:
         chdir: ~/plugins/oc-mirror/
-        target: build
-      register: make_build_output
+      register: podman_build_output
```
[the remark this answers is not in the document]
Comment (swapnil-bobade27) [reply playbooks/roles/ocp-cli-plugins/tasks/oc-mirror-cli-plugin.yml]: Removed the unused vars

**Review by yussufsh — APPROVED** [review]
/lgtm

**Pull request merged**